firmware/hw_layer/sensors/hip9011_logic.h

```diff
@@ -16,7 +16,7 @@
 #endif
 
 #define HIP9011_BAND(bore)		(900 / (CONST_PI * (bore) / 2))
-#define HIP9011_DESIRED_OUTPUT_VALUE	5.0f
+#define HIP9011_ANALOG_OUTPUT_MAX		5.0f
```

Comment:
huh how is this HIP change in this PR? should that be a separate PR?

Reply:
https://github.com/rusefi/rusefi/pull/4939
https://github.com/rusefi/rusefi/pull/4940